website/docs/d/resource_pool.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_resource_pool"
+sidebar_current: "docs-vsphere-data-source-resource-pool"
+description: |-
+  Provides a vSphere resource pool data source. This can be used to get the general attributes of a vSphere resource pool.
+---
+
+# vsphere\_resource\_pool
```

Comment:
_minor_ you don't actually need the backslashes here

Reply:
https://github.com/terraform-providers/terraform-provider-vsphere/pull/244#discussion_r153990508